postgres/datadog_checks/postgres/statements.py

```diff
@@ -181,6 +192,40 @@ def _get_pg_stat_statements_columns(self):
                 self._stat_column_cache = col_names
                 return col_names
 
+    def _check_called_queries(self):
+        pgss_view_without_query_text = self._config.pg_stat_statements_view
+        if pgss_view_without_query_text == "pg_stat_statements":
+            pgss_view_without_query_text = "pg_stat_statements(false)"
+
+        with self._check._get_main_db() as conn:
+            with conn.cursor() as cursor:
```

Comment:
```suggestion
            with conn.cursor(cursor_factory=CommenterCursor) as cursor:
```

Reply:
ah, I think this was a new change that wasn't present when I first started writing the PR so I just referenced existing connection handling. Easy enough to change.